File: category/sort/2750.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var n int
	var numlist []int
	fmt.Scanf("%d", &n)
	numlist = make([]int, n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &numlist[i])
	}

	sort.Slice(numlist, func(i, j int) bool {
		return numlist[i] < numlist[j]
	})

	for _, num := range numlist {
		fmt.Println(num)
	}
}

File: category/sort/2750.py
import sys

def main():
    n = int(sys.stdin.readline())
    numlist = []

    for i in range(0, n):
        numlist.append(int(sys.stdin.readline()))

    numlist.sort()

    for n in numlist:
        print(n)

    
    
if __name__ == "__main__":
    main()